http/CHANGELOG.md

```diff
@@ -2,6 +2,151 @@
 
 Changelog for `twilight-http`.
 
+## [0.5.0] - 2021-06-13
+
+### Upgrade Path
+
+Remove usage of `GetReactions::before`, `ErrorType::Formatting`,
+`ErrorType::Ratelimiting`, `CreateMessage::attachment`,
+`CreateMessage::attachments`, `CreateGuild::region`, `UpdateGuild::region`, and
+`Result`.
+
+Update usage of `CreateGuildFromTemplateError`, `CreateTemplateError`,
+`SearchGuildMembersError`, `UpdateTemplateError`.
+
+Replace the following usages:
+```diff
+-twilight_http::routing::Route::DeleteMessageSpecficReaction
++twilight_http::routing::Route::DeleteMessageSpecificReaction
+
+-twilight_http::request::channel::invite::CreateInvite::target_user
++twilight_http::request::channel::invite::CreateInvite::target_user_id
+
+-twilight_http::request::channel::invite::CreateInvite::target_user_type
++twilight_http::request::channel::invite::CreateInvite::target_type
+```
+
+`UpdateStageInstance` requests now look like this:
+
+```diff
+-client.update_stage_instance(channel_id, topic)?.await?;
++client.update_stage_instance(channel_id)
++    .topic(topic)?
++    .await?;
+```
+
+If previously using the `tracing` feature, and using a custom feature set, add
```

Comment:
To be clearer this only needs to be done if disabling the default features; using the features array is additive and leaving tracing out without disabling the defaults will still enable tracing

Reply:
so this can just be removed